config.toml.example

```diff
@@ -0,0 +1,633 @@
+# AggsenderPrivateKey is the private key which is used to sign certificates
+# ------------------------------------------------------------------------------
+AggSenderPrivateKey = {Path = "<PATH>", Password = "<PASSWORD>"}
+
+# ------------------------------------------------------------------------------
+# Proof Mode. Default is "AggchainProof" for FEP mode. Use "PessimisticProof"
+# for PP mode
+# ------------------------------------------------------------------------------
+Mode = "AggchainProof"
+CheckStatusCertificateInterval = "1s"
+
+# ------------------------------------------------------------------------------
+# MaxCertSize is the maximum size of the certificate
+# i.e (the emitted certificate cannot be bigger that this size)
+# 0 is infinite
+# ------------------------------------------------------------------------------
+# MaxCertSize = 0
+
+{{- if .use_agg_sender_validator }}
+{{- if eq .consensus_contract_type "pessimistic"}}
+RequireValidatorCall = true
+{{- else}}
+RequireValidatorCall = false
+{{- end}}
+[AggSender.ValidatorClient]
+URL = "aggkit-validator{{.deployment_suffix}}:{{.aggkit_validator_grpc_port}}"
+{{- end }}
+
+[AggSender.AggkitProverClient]
+UseTLS = false
+
+# ------------------------------------------------------------------------------
+# Config for Full Execution Proof chains
+# ------------------------------------------------------------------------------
+# [AggSender.OptimisticModeConfig]
+# SovereignRollupAddr = "<ADDR>"
+# TrustedSequencerKey = {Path = "<PATH>", Password = "<PASSWORD>"}
+# OpNodeURL = "<URL>"
+# RequireKeyMatchTrustedSequencer = true
+
+[AggSender.MaxSubmitCertificateRate]
+NumRequests = 20
+Interval = "1m"
+
+# ------------------------------------------------------------------------------
+# URLRPCL2 is the URL of the L2 RPC node
+# ------------------------------------------------------------------------------
+# URLRPCL2 = "<URL>"
+
+# ------------------------------------------------------------------------------
+# BlockFinality indicates which finality follows AggLayer accepted values are:
+# LatestBlock, SafeBlock, PendingBlock, FinalizedBlock, EarliestBlock
+# Default value is "LatestBlock"
+# ------------------------------------------------------------------------------
+BlockFinality = "LatestBlock"
+
+# ------------------------------------------------------------------------------
+# EpochNotificationPercentage indicates the percentage of the epoch
+# the AggSender should send the certificate
+# 0 -> Begin
+# 50 -> Middle
+# ------------------------------------------------------------------------------
+# EpochNotificationPercentage = 50
+
+# ------------------------------------------------------------------------------
+# AggSender will save the certificates to a file in this path
+# ------------------------------------------------------------------------------
+SaveCertificatesToFilesPath = "<PATH>"
```

Comment:
It seems like this config parameter is obsoleted in a meantime, as this is the only leftover [occassion](https://github.com/agglayer/aggkit/blob/47e3ab20368dedf466d5b8cebc2ba8aca8df3842/test/config/pp-config.toml.template#L203).